charts/add-ons/publicApi/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
This file requires to be cleaned up and completed like the other addons `Chart.yaml` files.